tokio/src/fs/read_dir.rs

```diff
@@ -55,6 +55,10 @@ enum State {
 
 impl ReadDir {
     /// Returns the next entry in the directory stream.
+    ///
+    /// # Cancellation safety
```

Comment:
I would suggest unifying the section name (`# Cancellation safety` vs `# Cancel safety`)

Reply:
Ah thanks I forgot
```suggestion
    /// # Cancel safety
```